### DISCUSSION TITLE
Separating folders and tags

### DISCUSSION THREAD

**Discussion by hectorfarahani** [standardnotes/forum#3124] [Suggestions]
- [x] **Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

**Is your feature request related to a problem?**
As a previous user of One Note, Evernote, Notion & Obsidian I found it hard to organize notes on the current implementation of tags & folders.

Normally, "Folder" is a hierarchal system and it is good for "grouping" things together. It sounds like categories.
But "Tag" is not hierarchal and we can use them freely to "mark" something.

I cannot say there is not any hierarchal way to organize notes, because I have folders, and I cannot say I am missing tags, because it is also there. But at the same time, I am missing both of them. It is confusing and not enough to organize notes.

Maybe for new users, it sounds fine and they will learn to use this kind of organizing/grouping system to organize their notes, but it is common in many note-taking apps and even other software and apps to have tags as a non-hierarchal system. E.G: Blogging tools.

**Describe the solution you'd like**
* Separated folders and tags
* Tags will not have sub-tags. 
* Folder will act as an actual folder or a category or a box where you can group or stack up notes.
* The UI of the Folders will not use the `#` sign. So it won't be mixed by tags even visually.
* Sub-folder doesn't sound necessary. Each note itself acts as a box with its content. (Folder > Note > Content). And I guess the effort of development will be reduced by ignoring/postponing sub-folders. Also, there will be room for user feedback about the new system of Folders and Tags.

Examples from other note-taking apps to put more sight on this feature request:
One Note: Notebook > Section > Page > Note + Tags.
Evernote: Notebook Stack > Notebook > Note + Tags.
 
The minimum requirement of the expected feature is like having `Folder > (Note + Tags)`
The current one sounds like `Folder-Tag > (Note + Folder-Tags)`

**Describe alternatives you've considered**
I had to use the existing feature whether as a Folder or as Tags. Missing a second organizing feature is sensible, the current UI/feature is confusing.

**Additional context**
Thanks for the existing features.

**Comment by fra-gadio** [comment]
I agree with the above with the exception that nested folders are a must for me and the reason I left evernote.
